python/taichi/lang/kernel_impl.py

```diff
@@ -574,6 +577,42 @@ def call_back():
                 callbacks.append(get_call_back(v, gpu_v))
         return tmp, callbacks
 
+    def get_paddle_callbacks(self, v, has_pp):
+        callbacks = []
+
+        def get_call_back(u, v):
+            def call_back():
```

Comment:
You can print here: `print(sys.getrefcount(v))`

Reply:
Because only in the cases of CPU-GPU/GPU-CPU the callback function be called. At present my develop environment does not test well.